owasp-top10-2017-apps/a1/mongection/src/app.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const mongoose = require('mongoose');
+const db = require('./db');
+
+const server = express();
+
+const PORT = 9995;
+
+server.use(bodyParser.json());
+
+server.use(function(request, response, next) {
+    response.header("Access-Control-Allow-Origin", "*");
```

Comment:
Ops! Just mentioning, by setting `Access-Control-Allow-Origin` as `*` a CSRF vulnerability may be added to your app. 😅

Reply:
I know this too, I forgot to change this :zipper_mouth_face: